src/Util/isWmsLayer.ts

```diff
@@ -0,0 +1,11 @@
+import Layer from 'ol/layer/Layer';
+import Source from 'ol/source/Source';
+import BaseTileLayer from 'ol/layer/BaseTile';
+import BaseImageLayer from 'ol/layer/BaseImage';
+import OlSourceImageWMS from 'ol/source/ImageWMS';
+import OlSourceTileWMS from 'ol/source/TileWMS';
+
+export function isWmsLayer(layer: Layer<Source>): layer is (BaseTileLayer|BaseImageLayer) {
+  const source = layer.getSource();
+  return source instanceof OlSourceImageWMS || source instanceof OlSourceTileWMS;
```

Comment:
See above.

Reply:
This is purely for typing. What were the problems? The only thing I can imagine is different versions of the library in the same build. If we work with types this would be the nicer way.

Reply:
I can't remember but the two different versions is the most likely issue. Not sure if you can allways avoid it 100% but as `ol` is just a peerdependcy we should give it a try.

Reply:
We had problems with `instanceof` checks while having `react-geo` as linked dependency in projects due to fact that two versions of ol would exist in the scope then. But we're able to fix this issue with an `alias` for ol. So I think this change is fine.